### ISSUE TITLE
Saving services & topics

### ISSUE THREAD

**Issue by cherrielam** [pg-irc/pathways-ux-ui#69]

**Comment by cherrielam** [comment]
Two versions of Bookmarking Services & Topics are proposed:
- v0.1: Both on the same page - [Preview in Sketch Cloud](https://sketch.cloud/s/a8Pkv/v/KvWbZe/a/1jqrGy/r/GmnbJ5/)
- v0.2: Separate tabs for services & topics - [Preview in Sketch Cloud](https://sketch.cloud/s/a8Pkv/v/KvWbZe/a/p1kJ9p/play)

**Comment by rasmus-storjohann-PG** [comment]
Neither solution looks particularly difficult to build. I don't have a strong preference for one or the other.

**Comment by aron-chen-pg** [comment]
It really depends on the use case, in which how many topics/services do we foresee the users to save. If we anticipate people saving lots of topics/services, having the separate tabs would definitely be more beneficial for the users in locating their saved topics/services.

**Comment by cherrielam** [comment]
@aron-chen-pg I tend to agree that v0.2 would feel more organized; is there a reason we wouldn't have one additional tab that says "All" and shows v0.1, and if the list is long, there would be a "show all (services)" button that takes you to the corresponding tab?
